### PR TITLE
Cart manage page

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- add : cart management page
- add: responsiveness 
- update links on home page

Screenshot 2022-02-17 at 7 55 32 PM

### CODE REVIEW

**Review by ooreally — APPROVED** [review]
Looks good to merge.

File: cart-management/cart.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.place-order-box {
+  margin: 1rem;
+  width: 22rem;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+.place-order-box .btn {
+  margin: 0.5rem 0rem 0.5rem 0rem;
```
Comment (Ashishgupta08) [inline cart-management/cart.css]: follow shorthand property - `margin: 0.5rem 0rem`

File: cart-management/cart.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.charges {
```
Comment (Ashishgupta08) [inline cart-management/cart.css]: try to use better naming conventions for classes

File: cart-management/cart.html [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+              <i class="bi bi-cart"></i>
+            </button>
+            <button class="btn-icon burger">
+              <i class="bi bi-list"></i>
+            </button>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="main-box">
```
Comment (Ashishgupta08) [inline cart-management/cart.html:72]: here you can have a class name like - `section-wrapper or container`

**Review by Ashishgupta08 — CHANGES_REQUESTED** [review]
UI looks really nice, just need to work on some sections so that it looks different from others.

Improve  naming conventions of class names 
Use shorthand properties
Fixed file and folder structure

**Pull request merged**